47.md

```diff
@@ -126,6 +127,224 @@ Response:
 Errors:
 - `PAYMENT_FAILED`: The payment failed. This may be due to a timeout, exhausting all routes, insufficient capacity or similar.
 
+### `multi_pay_invoice`
+
+Description: Requests payment of multiple invoices.
+
+Request:
+```jsonc
+{
+    "method": "multi_pay_invoice",
+    "params": {
+        "invoices": [
+          "lnbc50n1..." // bolt11 invoice
+        ],
+    }
+}
+```
+
+Response:
+
+For every invoice in the request, a separate response event is sent. To differentiate between the responses, each
+response event contains an `payment_hash` tag with the payment hash of the invoice it is responding to.
```

Comment:
this mentions `payment_hash` but is not in the response

Reply:
it is not in the response, it is in a `tag`